In [1]:
import pandas as pd
import sys
import os

# First, let's work on DE_Tdu_Tpr for short-liguled T. miscellus (Tms)

In [2]:
os.chdir("/ufrc/soltis/shan158538/TragFL_NewAnalysis/OutPut/DE_analysis/Tdu_Tpr_voom_Tms/")

In [6]:
df_for_Tms = pd.read_table("DE_Tdu_Tpr_min10_3rep.txt")

In [7]:
df_for_Tms.head()

logFC    ...            B
Tpr_TRINITY_DN11257_c2_g1|Tdu_TRINITY_DN16696_c... -0.053919    ...    -7.095424
Tpr_TRINITY_DN10844_c2_g7|Tdu_TRINITY_DN25328_c... -0.498146    ...    -4.727989
Tpr_TRINITY_DN15383_c4_g16|Tdu_TRINITY_DN14160_...  0.946122    ...    -3.370146
Tpr_TRINITY_DN1451_c0_g1|Tdu_TRINITY_DN16990_c3_g1 -0.602890    ...    -3.756732
Tpr_TRINITY_DN10853_c1_g4|Tdu_TRINITY_DN21860_c...  0.015868    ...    -7.296158

[5 rows x 6 columns]

In [9]:
df_for_Tms.shape

(8021, 6)

In [10]:
For_Tms_not_sig = df_for_Tms[df_for_Tms["adj.P.Val"] >= 0.05]

In [11]:
For_Tms_not_sig.head()

logFC    ...            B
Tpr_TRINITY_DN11257_c2_g1|Tdu_TRINITY_DN16696_c... -0.053919    ...    -7.095424
Tpr_TRINITY_DN10844_c2_g7|Tdu_TRINITY_DN25328_c... -0.498146    ...    -4.727989
Tpr_TRINITY_DN15383_c4_g16|Tdu_TRINITY_DN14160_...  0.946122    ...    -3.370146
Tpr_TRINITY_DN1451_c0_g1|Tdu_TRINITY_DN16990_c3_g1 -0.602890    ...    -3.756732
Tpr_TRINITY_DN10853_c1_g4|Tdu_TRINITY_DN21860_c...  0.015868    ...    -7.296158

[5 rows x 6 columns]

In [12]:
For_Tms_not_sig.shape

(6938, 6)

In [13]:
For_Tms_not_sig.to_csv("DE_Tdu_Tpr_not_sig_loci.txt")

In [14]:
For_Tms_sig = df_for_Tms[df_for_Tms["adj.P.Val"] < 0.05]

In [15]:
For_Tms_sig.head()

logFC    ...            B
Tpr_TRINITY_DN14551_c1_g11|Tdu_TRINITY_DN21735_...  1.608699    ...     0.069893
Tpr_TRINITY_DN11678_c3_g2|Tdu_TRINITY_DN24143_c... -3.856024    ...    -0.177865
Tpr_TRINITY_DN13298_c1_g1|Tdu_TRINITY_DN18431_c...  2.038795    ...    -0.073649
Tpr_TRINITY_DN10219_c6_g1|Tdu_TRINITY_DN19995_c... -1.509561    ...    -0.741504
Tpr_TRINITY_DN10516_c1_g3|Tdu_TRINITY_DN19561_c...  0.735129    ...    -1.617311

[5 rows x 6 columns]

In [16]:
For_Tms_sig.shape

(1083, 6)

In [17]:
For_Tms_sig.to_csv("DE_Tdu_Tpr_sig_loci.txt")

## For Tdu_Tpr_for_Tms, 6,938 loci are not signigicantly expressed between Tdu and Tpr; 1,083 loci showed differential expression between Tdu and Tpr

# Second, let's work on DE_Tdu_Tpr for long-liguled T. miscellus (Tml)

In [18]:
os.chdir("/ufrc/soltis/shan158538/TragFL_NewAnalysis/OutPut/DE_analysis/Tdu_Tpr_voom_Tml/")

In [19]:
df_for_Tml = pd.read_table("DE_Tdu_Tpr_min10_3rep.txt")

In [20]:
df_for_Tml.shape

(8248, 6)

In [21]:
For_Tml_not_sig = df_for_Tml[df_for_Tml["adj.P.Val"] >= 0.05]

In [22]:
For_Tml_not_sig.shape

(6886, 6)

In [23]:
For_Tml_not_sig.to_csv("DE_Tdu_Tpr_not_sig_loci.txt")

In [24]:
For_Tml_sig = df_for_Tml[df_for_Tml["adj.P.Val"] < 0.05]

In [25]:
For_Tml_sig.shape

(1362, 6)

In [26]:
For_Tml_sig.to_csv("DE_Tdu_Tpr_sig_loci.txt")

## For Tdu_Tpr_for_Tml, 6,886 loci are not signigicantly expressed between Tdu and Tpr; 1,362 loci showed differential expression between Tdu and Tpr